mysql/004-failover.md

```diff
@@ -2,19 +2,19 @@ Now, let's check that the data was persisted and is accessible on another node.
 
 Make a note of the location on which the volume got initially prosioned on:
 
-`storageos volume ls --format "table {{.Name}}\t{{.Size}}\t{{.Replicas}}\t{{.MountedBy}}\t{{.Location}}"`{{execute}}
+`kubectl exec -ti cli -n kube-system -- storageos get volume`{{execute}}
 
 Save the namespace/volume into a variable.
 
-`volume=$(storageos volume ls --format "table {{.Name}}\t{{.Replicas}}" | grep '0/0' | awk '{ print $1 }')`{{execute}}
+`volume=$(kubectl exec -ti cli -n kube-system -- storageos get volume | grep '0/0' | awk '{ print $2 }')`{{execute}}
 
 Create 1 replica for the volume, by adding a label to the volume.
 
-`storageos volume update --label-add storageos.com/replicas=1 $volume`{{execute}}
+`kubectl exec -ti cli -n kube-system -- storageos update volume replicas $volume 1`{{execute}}
 
 Check that the replica was created.
 
-`storageos volume ls --format "table {{.Name}}\t{{.Replicas}}"`{{execute}}
+`kubectl exec -ti cli -n kube-system -- storageos get volume`{{execute}}
 
 > Usually takes ~10 seconds to create the replica in this example.
 
```

Comment:
This is for the line below but there's no guarantee that the pods are going to be created on two separate nodes unless you use node selectors. That'd add some complexity that may or may not be required. Have a think about what you're trying to illustrate and decide what course to take.

Reply:
I tried it multiple times and it got deployed to a different node every time, I wonder if the person who wrote this (maybe @Arau ) knew that the scoring system of kube-scheduler would always select the other node. I will try to do it in a clearer/more consistent way though.

Reply:
Oh wait it does have a node selector actually!

Reply:
https://github.com/storageos/tutorials/blob/a38cc8d76159569349a36cec5f0bad526f9df79e/mysql/assets/mysql-pod2.yaml#L25

Reply:
Let's chat about it on slack!